apps/site/assets/js/event-page-setup.js

```diff
@@ -181,16 +181,17 @@ export default function() {
   document.addEventListener(
     "turbolinks:load",
     () => {
+      const viewPreviousEventsLink = document.querySelector(
+        ".m-view-previous-events"
+      );
+      viewPreviousEventsLink.classList.remove("hidden");
```

Comment:
Oh!!  Your question in the dev channel about getting attributes from the body - was it related to this PR?  [Cristen just taught me in the other JS PR](https://github.com/mbta/dotcom/pull/902#discussion_r621592263) that there is a CSS-only way that we can check to see if JS is enabled.  Although this seems to be a succinct solution as well :)